tools/modify_fsurdat/modify_template.cfg

```diff
@@ -56,13 +56,13 @@ lnd_lon_2 = 360
 # user-defined mask in a file, as alternative to setting lat/lon values
 landmask_file = UNSET
 
-# Non-crop PFT to be set to 100% according to user-defined mask.
-# If idealized = True and dom_nat_pft = UNSET, the latter defaults to 0
-# (bare soil). Valid values 0 to 14 (int).
-dom_nat_pft = UNSET
+# PFT/CFT to be set to 100% according to user-defined mask.
+# If idealized = True and dom_plant = UNSET, the latter defaults to 0
+# (bare soil). Valid values 0 to 78 (int).
```

Comment:
This is another magic number, although it's also only in a comment. And I think this might be similar to an issue that @negin513 and I discussed where we didn't see a great way to eliminate a magic number for a specific thing. So I'd like to discuss this with a bit...

Reply:
Maybe instead of specifying the values 0 and 78, I could send the user to the surface dataset and get the valid values there.

Reply:
One thing I did was setting `MAX_PFT` and `NAT_PFT` as constants in the code and used those constants, when referring to them both in comments and codes:

https://github.com/ESCOMP/CTSM/blob/04a0348573689d8d9bb9aa75fe3004adcacb787f/python/ctsm/site_and_regional/single_point_case.py#L21-L27

If you want, you can import these constants directly and reuse them.

Reply:
Thank you @negin513 !

In talking with @ekluzek we decided to let users figure out the valid range by using ncdump on their specific fsurdat file. This way we're not tied to any hardwired values.

Reply:
For example we still have fsurdat files with max_pft = 16 instead of 78

Reply:
Thanks for your response. Overall, @ekluzek and I also found it difficult to get rid of the hard-coding of 16 vs 78. But we tried it using the constants mentioned above. 

> For example we still have fsurdat files with max_pft = 16 instead of 78
In #1461, I put in explicit error checking to take care of this exact scenario. So if the user chose a file with 16 pfts on them and `dom_pft` is higher than 16 it gives an error.